pipeline/scripts/download-electron-mirror.js

```diff
@@ -8,22 +8,24 @@ const path = require('path');
 
 /*
 This script replaces existing electron & chromedriver modules
-with mirror dependencies specified by ELECTRON_MIRROR_VAR
-and ELECTRON_CUSTOM_DIR_VAR (see @electron/get). We use this 
+with mirror dependencies specified by ELECTRON_MIRROR_BASE_VAR
+and ELECTRON_MIRROR_CUSTOM_DIR_VAR, and `assetNumber`. We use this 
```

Comment:
Would it be accurate and useful to say something along the lines of "This script replaces existing electron & chromedriver modules with mirror dependencies specified by the ELECTRON_MIRROR_BASE_VAR and ELECTRON_MIRROR_CUSTOM_DIR_VAR environment variables. The values for these variables will be computed at build time based on the script-specified value of `assetNumber`"

Reply:
I have reworded some of the comments after offline chat